nemo_curator/utils/fuzzy_dedup_utils/io_utils.py

```diff
@@ -60,8 +60,9 @@ def _read_json_func(
 
 
 def get_text_ddf_from_json_path_with_blocksize(
-    input_data_paths, num_files, blocksize, id_column, text_column, input_meta
+    input_data_paths, num_files, blocksize, id_column, text_column, input_meta=None
 ):
+    print("Hello world!")
```

Comment:
Looks like this was left by mistake.

Reply:
Oops! 😅 Just removed it. Thank you for noticing it!